wafer/talks/admin.py

```diff
@@ -90,8 +90,9 @@ class TalkAdmin(CompareVersionAdmin):
 
     def get_queryset(self, request):
         qs = super(TalkAdmin, self).get_queryset(request)
+        review_aspect_count = ReviewAspect.objects.count()
         qs = qs.annotate(
-            review_count_annotation=models.Count('reviews'),
+            review_count_annotation=models.Count('reviews') / review_aspect_count,
```

Comment:
This doesn't seem right -- there should be only one review for each review, although there may be multiple scores for each review. Maybe something else has gone wrong or perhaps you could explain a bit more why you think there is a review for each aspect and not only a score for each aspect?